packages/block-editor/src/components/link-control/settings-drawer.js

```diff
@@ -28,10 +28,11 @@ function LinkSettingsDrawer( { children, settingsOpen, setSettingsOpen } ) {
 				className="block-editor-link-control__drawer-toggle"
 				aria-expanded={ settingsOpen }
 				onClick={ () => setSettingsOpen( ! settingsOpen ) }
-				icon={ settingsIcon }
-				label={ __( 'Link Settings' ) }
+				icon={ isRTL() ? chevronLeftSmall : chevronRightSmall }
 				aria-controls={ settingsDrawerId }
-			/>
+			>
+				{ __( 'Advanced' ) }
```

Comment:
I wonder whether we should add additional context for translators here with `_x`?

Reply:
Yeh that seems good.

Reply:
Added in https://github.com/WordPress/gutenberg/pull/51712/commits/ed450e503358c4e3477384729a2b92b7f835ff73